cpu/atmega_common/periph/gpio.c

```diff
@@ -244,54 +254,46 @@ static inline void irq_handler(uint8_t int_num)
     __exit_isr();
 }
 
-ISR(INT0_vect, ISR_BLOCK)
-{
+ISR(INT0_vect, ISR_BLOCK){
```

Comment:
Why did you change the style of unrelated code? Several other instances below.

Reply:
I missed this change. I ran uncrustify before pushing the branch. It does not seem to recognize ISR as a function and moved the curly brace therefore. I changed it back now.

Reply:
Looks like there is a fix being worked on for this issue: #9370